OJP/OJP_FareSupport.xsd

```diff
@@ -408,9 +432,9 @@
 					</xs:annotation>
 				</xs:element>
 			</xs:choice>
-			<xs:element name="EntitlementProduct" type="EntitlementProductRefStructure" minOccurs="0" maxOccurs="unbounded">
+			<xs:element name="EntitlementProducts" type="EntitlementProductListStructure" minOccurs="0" maxOccurs="unbounded">
```

Comment:
You are now introducing an explicit list element `EntitlementProducts` which will contain the `EntitlementProduct`s. Why does this element `EntitlementProducts` still have `maxOccurs="unbounded"`?
This would allow multiple `EntitlementProducts` lists beneath each other, each containing `EntitlementProduct` items.

These are the options to solve this:
1. Remove `maxOccurs="unbounded"` from `EntitlementProducts`. Now only one list may exist.
2. It is intentional that multiple `EntitlementProducts` elements each with `EntitlementProduct` items may exist, but then an explanation is missing why.
3. We do not need an explicit `EntitlementProducts` element to group the `EntitlementProduct` items at all. `EntitlementProduct` is not renamed to `EntitlementProducts` and kept with `maxOccurs="unbounded"`, only the type changes from `EntitlementProductRefStructure` to `EntitlementProductStructure`. `EntitlementProductListStructure` is removed and its annotation moved back to `EntitlementProduct`.

Reply:
Did 1. Hope this is ok for everybody